versions.props

```diff
@@ -1,26 +1,26 @@
 com.fasterxml.jackson.*:* = 2.11.3
-com.github.ben-manes.caffeine:caffeine = 2.8.8
+com.github.ben-manes.caffeine:caffeine = v2.8.8
```

Comment:
hmm, this seems off

```

FAILURE: Build failed with an exception.

* What went wrong:
A problem occurred configuring root project 'tritium'.
> Could not compute lock state from configuration 'unifiedClasspath' due to unresolved dependencies:
   * com.github.ben-manes.caffeine:caffeine:v2.8.8 (requested: 'com.github.ben-manes.caffeine:caffeine' because: requested)
        Failures:
           - Could not find com.github.ben-manes.caffeine:caffeine:v2.8.8.
  Searched in the following locations:
    - https://jcenter.bintray.com/com/github/ben-manes/caffeine/caffeine/v2.8.8/caffeine-v2.8.8.pom
    - https://dl.bintray.com/palantir/releases/com/github/ben-manes/caffeine/caffeine/v2.8.8/caffeine-v2.8.8.pom
    - https://dl.bintray.com/marshallpierce/maven/com/github/ben-manes/caffeine/caffeine/v2.8.8/caffeine-v2.8.8.pom

   * com.github.ben-manes.caffeine:caffeine:v2.8.8 (requested: 'com.github.ben-manes.caffeine:caffeine:v2.8.8' because: constraint)
        Failures:
           - Could not find com.github.ben-manes.caffeine:caffeine:v2.8.8.
  Searched in the following locations:
    - https://jcenter.bintray.com/com/github/ben-manes/caffeine/caffeine/v2.8.8/caffeine-v2.8.8.pom
    - https://dl.bintray.com/palantir/releases/com/github/ben-manes/caffeine/caffeine/v2.8.8/caffeine-v2.8.8.pom
    - https://dl.bintray.com/marshallpierce/maven/com/github/ben-manes/caffeine/caffeine/v2.8.8/caffeine-v2.8.8.pom
```